src/matchHistory/MatchHistory.tsx

```diff
@@ -28,7 +29,9 @@ export const MatchHistory = React.memo(function MatchHistory() {
                 alignItems: 'center',
             }}
         >
-            <h1>Match History</h1>
+            <Heading color='bodyFont' padding='6'>

```

Comment:
![image](https://user-images.githubusercontent.com/952050/194369394-f9fbca51-6875-4ac1-a079-f63b243558ff.png)

I know this is not something you broke. however, if you are already playing with styling, this may be something you can fix in the meantime. I can always go in later to fix this.

Reply:
will do in a future pr